cypress/integration/e2e/test8.spec.ts

```diff
@@ -0,0 +1,89 @@
+import Login from '../../pageObjects/login'
+import Purchase from '../../pageObjects/purchase'
+import Signin from '../../pageObjects/signin'
+import Address from '../../pageObjects/address'
+import Orders from '../../pageObjects/orders'
+
+
+
+describe('End to End Scenario', function()  {
```

Comment:
when you check the new README.md file, you will realise the need to have more clear names for the test spec files.

There is a section like this. Note: The names of the scenarios need not be exact same - since these are file names in your case but you would definitely need to come up with better names for the files.

## About the tests in this repository

  This repository contains the following Selenium tests:

  - End to End Scenario
  - Login as username
  - Login as Locked User
  - Offers for mumbai geo-location
  - Apply Apple Vendor Filter
  - Apply Lowest to Highest Order By
  - Login as User with no image loaded
  - Login as User with existing Orders

  Note: "End to End Scenario" is the default test in all the single test run profiles. This test also demonstrates the [Page Object Model design pattern](https://www.browserstack.com/guide/page-object-model-in-selenium).